src/HouGeo.cpp

```diff
@@ -591,14 +600,21 @@ namespace houio
 				{
 					std::string string = stringsArray->get<std::string>( i );
 					attr->strings.push_back(string);
-					//qDebug() << QString::fromStdString(string);
 				}
 				attr->numElements = numElements;
+			}
 
-				attr->m_name = attrName;
-				attr->m_type = attrType;
-				//attr->storage = attrStorage;
-				attr->tupleSize = 1;
+			if( attrData->hasKey("indices") )
+			{
+				json::ObjectPtr indicesObject = toObject(attrData->getArray("indices"));
+				json::ArrayPtr indices = indicesObject->getArray("rawpagedata");
+				int numElements = indices->size();
+				for( int i=0;i<numElements;++i )
+				{
+					int index = indices->get<int>(i);
```

Comment:
We could declare index before the loop in order to order new allocation / destruction at each iteration

Reply:
Done in next a2e8aa10fdbdc70a5c33bf5743955177b94c12b2